helm/templates/servicemanager/statefulset.yml

```diff
@@ -56,66 +56,136 @@ spec:
       {{- end }}
       containers:
         - name: pinot-servicemanager
-          image: {{ include "pinot.servicemanager.image" .}}
+          image: {{ include "pinot.image" $ }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args: [
             "StartServiceManager",
             "-clusterName", {{ .Values.cluster.name | quote }},
             "-zkAddress", {{ include "zookeeper.path" . | quote }},
             "-port", {{ .Values.servicemanager.port | quote }},
+            "-bootstrapServices",
+            "CONTROLLER",
+            "BROKER",
+            "SERVER",
+            {{- if .Values.servicemanager.minion.enabled }}
```

Comment:
do we need minion on single node cluster?

Reply:
If we have only real time tables we don't need